src/modules/Elsa.Caching/Extensions/ModuleExtensions.cs

```diff
@@ -0,0 +1,36 @@
+using Elsa.Caching.Features;
+using Elsa.Features.Services;
+
+// ReSharper disable once CheckNamespace
+namespace Elsa.Extensions;
+
+/// <summary>
+/// Provides methods to install and configure the distributed caching feature.
+/// </summary>
+public static class ModuleExtensions
+{
+    /// <summary>
+    /// Adds the <see cref="DistributedCacheFeature"/> feature to the system.
+    /// </summary>
+    public static MemoryCacheFeature UseMemoryCache(this IModule module, Action<MemoryCacheFeature>? configure = default)
+    {
+        return module.Configure(configure);
+    }
+    
+    /// <summary>
+    /// Adds the <see cref="DistributedCacheFeature"/> feature to the system.
+    /// </summary>
+    public static DistributedCacheFeature UseDistributedCache(this MemoryCacheFeature memoryCacheFeature, Action<DistributedCacheFeature>? configure = default)
```

Comment:
It could be part of that, but then we'd have to duplicate the feature for an e.g. Redis implementation. Instead, we should add a shared Elsa.Caching.Distributed project that captures the distributed caching primitives and services, that is then shared by MassTransit and e.g. Redis and potentially others.

Reply:
Having a separate Distributed project for this might be worth it when this project grows in size. For now we can leave it here.

Reply:
Just noticed you already moved it :)